simpleclient_hotspot/src/main/java/io/prometheus/client/hotspot/StandardExports.java

```diff
@@ -56,8 +58,19 @@ public StandardExports() {
   public List<MetricFamilySamples> collect() {
     List<MetricFamilySamples> mfs = new ArrayList<MetricFamilySamples>();
 
-    mfs.add(new CounterMetricFamily("process_cpu_seconds_total", "Total user and system CPU time spent in seconds.",
-        osBean.getProcessCpuTime() / NANOSECONDS_PER_SECOND));
+    try {
+      Method method = osBean.getClass().getMethod("getProcessCpuTime", null);
+      if (!method.isAccessible()) {
```

Comment:
Sorry - brainfreeze.
The class has default access, that's why the method needs to be set accessible: https://github.com/frohoff/jdk8u-jdk/blob/master/src/solaris/classes/sun/management/OperatingSystemImpl.java - naturally I don't have Oracle sources available as a reference on Internet - but same there if you step debug into it.